src/pages/home/report/ReportActionItem.js

```diff
@@ -676,7 +676,7 @@ function ReportActionItem(props) {
         >
             <Hoverable
                 shouldHandleScroll
-                disabled={Boolean(props.draftMessage)}
+                isDisabled={Boolean(props.draftMessage)}
```

Comment:
I think we should leave it as it was. See: https://reactnative.dev/docs/keyboardavoidingview#enabled

`isFoo` is a good name for a `bool` when it states a fact that something "is foo" (after the fact, we could say). When the boolean acts as a command for something "to become foo" or "to be foo", `shouldFoo` would be better, but existing conventions should also be taken into consideration.

As the React Native core has an `enabled` parameter in a built-in class, we should stick with `disabled`, I think.

Naming is hard.

Reply:
While I agree with approach mentioned by you, there was a [request to rename `disabled` to `isDisabled](https://github.com/Expensify/App/pull/31518#discussion_r1409682612). 

Please let me know whati s the final decision 👍

Reply:
@roryabraham Any thoughts? I have some rationale for sticking with `disabled`, and I don't think we have a solid convention for sticking with `isFoo` naming for component parameters or function arguments.

But on the other hand, I don't have strong feelings about it.

Reply:
Yeah, let's please call this `isDisabled`. This convention is clearly outlined in our style guide:

> Boolean props or variables must be prefixed with should or is to make it clear that they are Boolean. Use should when we are enabling or disabling some features and is in most other cases.

I don't think `shouldDisable` or `shouldBeDisabled` is as clear, so `isDisabled` is good 👍🏼